prov/efa/src/rxr/rxr_read.h

```diff
@@ -68,7 +68,8 @@ enum rxr_read_context_type {
 enum rxr_read_entry_state {
 	RXR_RDMA_ENTRY_FREE = 0,
 	RXR_RDMA_ENTRY_CREATED,
-	RXR_RDMA_ENTRY_PENDING
+	RXR_RDMA_ENTRY_PENDING,
+	RXR_RDMA_ENTRY_SUBMITTED,
```

Comment:
Doesn't the `RXR_RDMA_ENTRY_PENDING` state need to be set when the entry is added to `read_pending_list`?

Reply:
Yes, the new state ```RXR_RDMA_ENTRY_SUBMITTED``` is set when the entry is removed from ```read_pending_list```.

Reply:
My point is that `RXR_RDMA_ENTRY_PENDING` is never set which will leave it on the list if `rxr_read_handle_error` is called when it is in that state.

Reply:
Nevermind, I see it now. I thought this was a new state.